static/src/stylesheets/module/identity/_identity.scss

```diff
@@ -384,6 +384,46 @@
 
 /* Saved-content
    ========================================================================== */
+.saved-contents {
+    .fc-item__meta {
+        @include mq($until: desktop) {
+            position: relative;
+        }
+
+        @include mq(desktop) {
+            position: absolute;
+            top: 0;
+        }
+    }
+
+    .fc-item__content,
+    .fc-item--has-metadata .fc-item__content {
+        @include mq($until: desktop) {
+            padding-bottom: $gs-gutter * 2;
+        }
+    }
+
+    .saved-content__button {
+        @include mq($until: desktop) {
+            right: $gs-gutter / 2;
+        }
+
+        @include mq(desktop) {
+            right: 0;
+            top: 8px;
+        }
+    }
+
+    .fc-item--has-cutout .saved-content__button {
+        @include mq(tablet) {
+            right: -130px;
```

Comment:
if these are based on hard coded amounts elsewhere, add a comment explaining that and pointing back. probably also a good idea to leave a comment in the original place too